src/index.css

```
@@ -170,6 +170,94 @@ html:not(.dark) ::-webkit-scrollbar-thumb {
+
+.premium-cursor-trail {
+  width: 34px;
+  height: 34px;
+  margin: -17px 0 0 -17px;
+  border: 1px solid rgba(99, 102, 241, 0.28);
+  background: radial-gradient(circle, rgba(99, 102, 241, 0.18), transparent 65%);
+  filter: blur(0.2px);
+  animation: cursorReveal 500ms ease-out forwards;
+}
+
+@keyframes cursorReveal {
```

Stylelint (17.13.0)

[error] Expected keyframe name "cursorReveal" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)